plugins/bcli.c

```diff
@@ -277,7 +277,7 @@ static void next_bcli(enum bitcoind_prio prio)
 	if (!bcli)
 		return;
 
-	bcli->pid = pipecmdarr(NULL, &bcli->fd, &bcli->fd,
+	bcli->pid = pipecmdarr(NULL, &bcli->fd, &pipecmd_preserve,
```

Comment:
I'm not able to see what the `pipecmd_preserve` is in the Github UI, but from the Github actions, U assumes that we need to check also the value of `pipecmd_preserve` in case we will request a block height not in the blockchain yet?